docs/DeveloperGuide.md

```diff
@@ -1484,17 +1474,30 @@ Assumptions: The sample data provided by InternBuddy is used, where there is a t
 
 <div style="page-break-after: always;"></div>
 
-## **Appendix C: Proposed Design Tweaks for Feature Flaws**
+## **Appendix C: Planned Enhancements**
 While we strive to make InternBuddy a perfect product for you, there are nevertheless areas of improvement.
 Hence, we are constantly striving to make InternBuddy better. This appendix documents some of the areas
 that our team is currently working on, and we would release them in the future once their functionalities
 have been finalised.
 
 ### Title of Design Tweak
 **Problem:**
-Explain the problem here
+Prefixes are case-sensitive, hence command arguments such as ` T/` will be interpreted as plain text. For example, `add n/Visa r/Software Engineer s/New d/2023-03-01 t/Golang T/c++` will add an internship entry with company name
+   `Visa`, role `Software Engineer`, status `New`, deadline of application `2023-03-01`, 
+   and tag `Golang T/c++`.
+  Therefore, it is possible to add substrings such as `T/`, `C/` or `R/` to any of the fields, even though the user could have intended to enter `t/`, `c/` or `r/`. 
+
+Moreoever, commmands such as `find` and `delete-field` do not consider prefixes such as `c/`, although other commands such as `edit` and `add` do  use the prefix `c/`. In the case of `find` and `delete-field`, it is possible to add the substring `c/` to any of the fields, even though the user could have intended to enter `c/` as a command prefix.
+
+We originally intended for this design to allow maximum flexibility for the user. If the user had entered something wrongly, it is possible to just use the command `edit`. However, this design could have an unintentional side-effect.
+
+ For example, using the `find` command, `find t/numbers t/cheese` tries to find entries with either the tag `numbers` or `cheese`. Tag matching is case-insensitive, so it also tries to find `Numbers` or `CheEse`. Similarly, `delete-field t/numbers t/cheese` tries to delete entries with either the tag `numbers` or `cheese` or `Numbers` or `CheEse`
```

Comment:
Do you think it's more appropriate to use tags that are related to our project instead of `cheese`?